johnwayodi/store-manager
# #161145157  Add Attendant Profile Page

#### What does this PR do?
Adds view where the Store Attendant can view their profile which contains the total number of products attendant has sold, the total cost of the products and the total number of sale record created

#### Description of Task to be completed?
Ensure that the correct values are loaded into the fields for each individual store attendant

#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/store-manager.git`
2. Ensure you are in the directory where you cloned this repository, store-manager by default
3. `git checkout ft-attendant-profile-161145157`
4. open attendant-profile.html on browser

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#161145157

#### Screenshots (if appropriate)
![attendant-profile](https://user-images.githubusercontent.com/10683903/46797874-01284580-cd59-11e8-8192-91e43ff3fafd.png)

#### Questions
Any feedback on what I can improve or change? Drop a comment or suggestion below
